src/main/java/org/datadog/jenkins/plugins/datadog/clients/DatadogHttpClient.java

```diff
@@ -347,8 +355,13 @@ public boolean validate() throws IOException, ServletException {
         } catch (Exception e) {
             if (conn != null && conn.getResponseCode() == HTTP_FORBIDDEN) {
                 logger.severe("Hmmm, your API key may be invalid. We received a 403 error.");
```

Comment:
For consistency, should we move this human readable line below the stackTrace one? (In a couple other places too)

Reply:
I don't think it's necessary here.

Reply:
It's consistent with https://github.com/jenkinsci/datadog-plugin/pull/24/files/cbf59640113264bf64dc9528b8d001e46a1ccfde#diff-daeae67ef2298d24148f5336f3887d7eL307

Reply:
Its not consistent with all of the other log lines though - https://github.com/jenkinsci/datadog-plugin/pull/24/files/cbf59640113264bf64dc9528b8d001e46a1ccfde#diff-daeae67ef2298d24148f5336f3887d7eR364 😄